afm/logging.py

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+#
+import logging
+import json_log_formatter
+import time
+
+logger = logging.getLogger('arrow-flight-module')
+app_uuid = ''
+
+class FybrikFormatter(json_log_formatter.JSONFormatter):
+    def json_record(self, message: str, extra: dict, record: logging.LogRecord) -> dict:
+        extra['message'] = message
+        extra['level'] = record.levelname
+        extra['name'] = record.name
+        extra['file'] = record.filename
+        extra['lineno'] = record.lineno
+        extra['funcName'] = record.funcName
+        extra['time'] = time.ctime(record.created)
+        extra['app-uuid'] = app_uuid
```

Comment:
The key should be the same constant as used in the manager.  const FybrikAppUUID = "app.fybrik.io/app-uuid".  Also, note the other constants listed in pkg/logging/logging.go in the manager.  These are also keys for parameters that should be included in the the logs where appropriate.